docs/cookbook/creating_content_with_binary_attachments_via_rest_api.md

```diff
@@ -0,0 +1,147 @@
+# Creating content with binary attachments via REST API
+
+## Description
+
+This recipe illustrates how to create content via the REST API on the example of uploading an image file.
+
+## Solution
+
+The creation of the draft is done with a POST request on /api/ezp/v2/content/objects
+with a body containing the required and structured data.
+As for the response, it's possible to use either JSON or XML in input.
+The following examples use JSON.
```

Comment:
This snippet assumes that HTTP Basic Auth is enabled.  https://doc.ezplatform.com/en/1.11/api/general_rest_usage/#http-basic-authentication